src/anyp/Uri.h

```diff
@@ -104,6 +104,9 @@ class Uri
     /// the static '*' pseudo-URI
     static const SBuf &Asterisk();
 
+    /// %-encode characters in a buffer
+    static SBuf Encode(const SBuf &, const CharacterSet &);
```

Comment:
It is not clear what the second parameter means -- "bad" characters to encode or "good" characters to keep as is.

Reply:
"only" characters

Reply:
Please allow me to rephrase this change request:

Please adjust the method description and/or declaration to document the meaning of the second parameter. That meaning is currently impossible to reliably discern from the method interface and its documentation.

I recommend addressing this change request _after_ https://github.com/squid-cache/squid/pull/617#discussion_r417424103.

Reply:
Should be resolved in 7fc9da84b4ca3f57fa6190f369098fdd6a9313b4